package.json

```diff
@@ -54,5 +54,9 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "prettier": "^3.0.0",
     "typescript": "^5.1.6"
+  },
+  "dependencies": {
```

Comment:
I think these dependencies should be specified under `peerDependencies` rather than `dependencies` because a project that uses this library also needs to install these dependencies